contentcuration/contentcuration/models.py

```diff
@@ -276,9 +275,9 @@ def check_channel_space(self, channel):
         active_size = float(active_files.aggregate(used=Sum('file_size'))['used'] or 0)
 
         staging_tree_id = channel.staging_tree.tree_id
-        channel_files = self.files.select_related('contentnode')\
+        channel_files = self.files\
                             .filter(contentnode__tree_id=staging_tree_id)\
-                            .values('checksum', 'file_size')\
+                            .values('checksum')\
```

Comment:
It looks like the `file_size` field is still used when calculating `staged_size`. Is it okay to remove this?

Reply:
Yes! The values call only affects the subsequent distinct call, so the only slight thing this does is disregard if the same checksum with different filesizes - but also makes sure that we are only doing a distinct by the checksum (whereas before we were doing a `DistinctSum` on the file_sizes, which would disregard different checksums with the same file_size).

Reply:
Ah, right, thanks! :)